docs/reference/executionEngines.md

```diff
@@ -4,16 +4,18 @@ title: Execution Engines
 ---
 
 An execution engine is a technology/library used by SDLB to transform data. SDLB supports different execution engines and is able to combine different execution engines in the same data pipeline / job.
-Execution engines can be categorized based on the data structure they provide transport and transform data. The transport object of data between DataObjects and Actions is called a SubFeed.
+The data structure used to transport data between DataObjects and Actions is called a SubFeed.

```

Comment:
FYI I hope you can live with that new description. A bit less precise, but easier to understand